rivalcfg/handlers/rgbgradientv2.py

```diff
@@ -0,0 +1,269 @@
+
+----
+
+A Python ``dict`` can also be used (Python API only)::
+
+    {
+        "duration": 1000,  # ms
+        "colors": [
+            {"pos": 0, "color": "red"},
+            {"pos": 33, "color": "#00FF00"},
+            {"pos": 66, "color": (0, 0, 255)},
+        ]
+    }
+
+.. NOTE::
+
+   A maximum of 14 color stops can be defined in a gradient.
+
+
+Device Profile
+--------------
+
+Example of a rgbgradient value type in a device profile:
+
+::
+
+    profile = {
+
+        # ...
+
+        "settings": {
+
+            "logo_color": {
+                "label": "Logo LED colors and effects",
+                "description": "Set the colors and the effects of the logo LED",
+                "cli": ["-c", "--logo-color"],
+                "report_type": usbhid.HID_REPORT_TYPE_FEATURE,
+                "command": [0x05, 0x00],
+                "value_type": "rgbgradientv2",
+                "rgbgradient_header": {
+                    "header_length": 28,       # Length of the header excuding command / LED ID
+                    "led_id_offsets": [0, 5],  # Offset of the "led_id" fields
+                    "duration_offset": 6,      # Offset of the "duration" field
+                    "duration_length": 2,      # Length of the "duration" field (in Bytes)
+                    "repeat_offset": 22,       # Offset of the "repeat" flag
+                    "triggers_offset": 23,     # Offset of the "triggers" field (buttons mask)
+                    "color_count_offset": 27,  # Offset of the "color_count" field
+                },
+                "led_id": 0x01,
+                "default": "rgbgradient(duration=1000; colors=0%: #ff0000, 33%: #00ff00, 66%: #0000ff)",
+            },
+
+        },
+
+        # ...
+
+    }
+
+
+CLI
+---
+
+Example of CLI option generated with this handler::
+
+   -c LOGO_COLOR, --logo-color LOGO_COLOR
+                         Set the colors and the effects of the logo LED (default:
+                         rgbgradient(duration=1000; colors=0%: #ff0000, 33%: #00ff00, 66%: #0000ff))
+
+Example of CLI usage::
+
+    rivalcfg --logo-color="rgbgradient(duration=1000; colors=0%: #ff0000, 33%: #00ff00, 66%: #0000ff)"
+    rivalcfg --logo-color=red
+    rivalcfg --logo-color=FF1800
+
+
+Functions
+---------
+"""  # noqa
+
+
+from ..helpers import uint_to_little_endian_bytearray, merge_bytes
+from ..helpers import bytes_to_high_low_nibbles, nibbles_to_byte
+from ..color_helpers import is_color
+from .rgbgradient import _handle_color_tuple, _handle_color_string
+from .rgbgradient import _handle_rgbgradient_dict, _handle_rgbgradient_string
+from .rgbgradient import is_rgbgradient, CheckGradientAction
+
+
+# Compatibility with Python 2.7.
+# On Python 2 the type is 'unicode'
+# On Python 3 the type is 'str'
+_unicode_type = type(u"")
+
+_default_duration = 1000
+
+
+def process_value(setting_info, colors):
+    """Called by the :class:`rivalcfg.mouse.Mouse` class when processing a
+    "rgbcolor" type setting.
```

Comment:
```suggestion
    "rgbgradientv2" type setting.
```

I will check `rgbgradient` too for this one, there is probably a mistake there too xD